compute_tools/src/compute.rs

```diff
@@ -878,6 +879,11 @@ impl ComputeNode {
         // 'Close' connection
         drop(client);
 
+        if let Some(ref local_proxy) = spec.local_proxy_config {
+            info!("configuring local-proxy");
```

Comment:
Seems to be some discrepancies with `local-proxy` and `localproxy`

Reply:
I'll switch everything over to snake case.